web/Cargo.toml

```diff
@@ -52,12 +52,12 @@ chrono = { version = "0.4", default-features = false, features = ["wasmbind", "c
 getrandom = { version = "0.2", features = ["js"] }
 serde = { version = "1.0.195", features = ["derive"] }
 thiserror = "1.0"
-base64 = "0.21.5"
+base64 = "0.21.7"
 async-channel = "2.1.1"
 futures-util = { version = "0.3.30", features = ["sink"] }
 gloo-net =  { version = "0.5.0", default-features = false, features = ["websocket"] }
 #TODO: change when fix released
```

Comment:
Is this comment still needed?

Reply:
Nope, good point, it hasn't been since the end of October. :)